atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TargetedSweepEteTest.java

```diff
@@ -106,6 +105,7 @@ public void targetedSweepCleansUpUnmarkedStreamsTest() {
     }
 
     @Test
+    @Ignore // TODO (jkong): This is obviously not the desired behaviour, but we are doing this for safety.
```

Comment:
No need to ignore, just kill this test - it documents the broken behaviour at the time.